bindings/python/CMakeLists.txt

```diff
@@ -17,7 +17,8 @@ project(python)
 execute_process(
   COMMAND "${Python_EXECUTABLE}" -m nanobind --version
   OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE NB_VERSION)
-if(NB_VERSION VERSION_GREATER_EQUAL 1.8.0)
+# we are fine with 2.0.0
+if(NB_VERSION VERSION_GREATER_EQUAL 2.0.0)
```

Comment:
Should we just go with 2.1.0 across the board?

Reply:
Some distros may have 2.1.0? Not sure about if we should support that.

Reply:
Okay, so 2.0 is significantly more widely available than 2.1? This is fine if so - I was just thinking we could reduce CMake complexity a bit.